packages/backend/src/open_payments/payment/incoming/routes.ts

```diff
@@ -70,7 +70,8 @@ async function getIncomingPaymentPublic(
 ) {
```

Comment:
In the `listIncomingPayments` function, we need to make sure to pass in `ctx.params.tenantId` in the `deps.incomingPaymentService.getWalletAddressPage`.

(something like [this](https://github.com/interledger/rafiki/pull/3171/commits/799026a362166aba563031e9862bb2caa41c9940#diff-789c9d4089f6ed3faac07bb57227080d576bb0d9f24d8eb9dafac9498a5efbf5R153-R159))